test/integration/api/delete_account_secrets.sh

```diff
@@ -10,5 +10,5 @@ fi
 
 for secret in $(oc get secrets -n "${NAMESPACE}" | awk "/${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD}/"'{ print $1 }')
 do
-    oc delete secret $secret -n ${NAMESPACE}
-done
\ No newline at end of file
+    oc delete secret $secret -n ${NAMESPACE} || true
```

Comment:
What is this doing? My brain is having trouble parsing this

Reply:
It's a failsafe.  It deletes the secret and if that fails (typically due to the secret not being present) it goes into the `|| true` and just continues on with execution.

Specifically I have a race-condition on my machine where sometimes this script is called while the secrets are being deleted with the accountclaim, so the for loop will pick up the secrets but the secret will be gone by the time the `oc delete` call is run.

Reply:
Got it 👍